### ISSUE TITLE
Compatible with KP115? 

### ISSUE THREAD

**Issue by CassSD** [jamesbarnett91/tplink-energy-monitor#77]
I'm considering buying TP Link KP115 with energy monitoring, which is apparently a replacement of HS110. Does anyone know if this programme is compatible with KP115? I'm new to smart home devices, and seeking a good real time energy monitoring system. I will use a UK plug if relevant. Cheers!

**Comment by frostinel** [comment]
> I'm considering buying TP Link KP115 with energy monitoring, which is apparently a replacement of HS110. Does anyone know if this programme is compatible with KP115? I'm new to smart home devices, and seeking a good real time energy monitoring system. I will use a UK plug if relevant. Cheers!

Yes it works. Just got one yesterday. So far so good. Same behaviour, but I like it is a bit smaller. Hope it lasts like the other ones.

**Comment by CassSD** [comment]
Thanks for your quick reply! Really encouraging to know!

**Comment by FreakNL** [comment]
It does not with the P115, sadly :(

**Comment by andrunl** [comment]
I can confirm that it works with Kp115 and with the latest EP25.
